Lib/idlelib/help.py

```
@@ -176,31 +176,39 @@ def bind_events(self):
         shortcut = 'Command' if darwin else 'Control'
         # Zoom out works with or without shift.
         self.widget.event_add(
-                '<<zoom-text-out>>',
+                '<<zoom-text-in>>',
```

I guess the pseudoevent error was masked since the following line has the same error and the direction was recalculated (in former version of zoom).

---

While refamiliarizing myself with this code, I kept getting confused on what 'zoom out' and 'zoom in' meant and added a comment to the new methods in help.py to say which means larger text and which means smaller text.  Once I did that, I realized I had it backwards in the definitions.

---

I agree that these names are potentially confusing and propose above (May 17 comment on line 180) 'make_font_bigger' and 'make_font_smaller'.